### ISSUE TITLE
@default(@dbgenerated(...)) generates [object Object]; fails

### ISSUE THREAD

**Issue by cnjmike** [luisrudge/prisma-generator-fake-data#19]
```
model Example { 
someId Int @default(dbgenerated("(current_setting('app.clinic_id'::text)::integer)"))

```

generates into fake-data.ts:

```
someId: [object Object]
```

which errors out:

```
 fake-data.ts:210:16 - error TS2552: Cannot find name 'object'. Did you mean 'Object'?

    210     someId: [object Object],
                       ~~~~~~

      node_modules/typescript/lib/lib.es5.d.ts:270:13
        270 declare var Object: ObjectConstructor;
                        ~~~~~~
        'Object' is declared here.
```

Adding @db.Integer to the prisma schema line has no effect.

I also see an example of a String attribute with @dbgenerated that generates:

```
someId: '[object Object']
```

which does not error but is not helpful.

I think the issue is in createMethod.js:37 and :40, where $field.default is returning [object Object].  No handling for dbgenerated inside default?

Thanks

**Comment by luisrudge** [comment]
yeah, it doesn't support `dbgenerated` as a default value at the moment. In your specific case, the issue is here: https://github.com/luisrudge/prisma-generator-fake-data/blob/main/src/helpers/createMethods.ts#L45-L47

I guess it's closely related to https://github.com/luisrudge/prisma-generator-fake-data/issues/17 and https://github.com/luisrudge/prisma-generator-fake-data/issues/12. For unsupported types or values, it'd be great to allow the developer to override the default behaviour and just add `FAKE: whatever_you_want_here()`
